packages/web-app/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "nation3-court-app",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
```

Comment:
`cd packages/web-app/`
`yarn build` fails to compile when I try it in this package.

```javascript
./components/ParticipantRow.tsx
24:13  Warning: Unexpected any. Specify a different type.  @typescript-eslint/no-explicit-any
26:18  Warning: Unexpected any. Specify a different type.  @typescript-eslint/no-explicit-any
30:18  Error: Unexpected empty arrow function.  @typescript-eslint/no-empty-function
```

Reply:
I'm not building it yet. It's on strict mode so any typescript error would prevent the build. This is going to be amended before merge.